source/draw/RasterBgr.ooc

```diff
@@ -117,7 +117,7 @@ RasterBgr: class extends RasterPacked {
 		x, y, imageComponents: Int
 		requiredComponents := 3
 		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntSize2D new(x, y))
+		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents, true), IntSize2D new(x, y))
```

Comment:
By default this is set to false which means the data is not owned by the buffer. We want the returned image own the allocated data. It will leak otherwise.